components/molecule/phoneInput/src/index.scss

```diff
@@ -37,15 +37,40 @@
 
         & > input {
           background: none;
+
+          &.sui-AtomInput-input--noBorder {
+            position: relative;
+          }
```

Comment:
is it ok to be overriding styles this way? 🤔 shouldn't we allow for `atom/input` itself to behave as we want when required?

Reply:
What change would you propose? 😄 
The issue is that `MoleculeInputField` wasn't designed to work within the context of `MoleculePhoneInput`, and its behaviour varies within this context. On its own, `MoleculeInputField` works well; it's a single input and occupies the same space, so they move in harmony. However, within `MoleculePhoneInput`, there are additional components that break this behaviour. So, I understand that `MoleculeInputField` shouldn't be aware of the issues that `MoleculePhoneInput` might cause. 😄

Reply:
> So, I understand that `MoleculeInputField` shouldn't be aware of the issues that `MoleculePhoneInput` might cause. 😄

of course I don't mean any coupling or making `MoleculeInputField` aware of any wrapper / consumer. My point has more to do with potential problems down the road with this approach of overriding styles. If there's no other way around, fine, but we might find unexpected styles when theming for `MoleculeInputField`.